types/ramda/tsconfig.json

```diff
@@ -154,7 +154,6 @@
         "test/nthArg-tests.ts",
         "test/o-tests.ts",
         "test/objOf-tests.ts",
-        "test/omit-tests.ts",
```

Comment:
This file was modified, but dropped from the array; not sure if that's intentional but it'll mean the tests won't get run. (We don't currently check for unreferenced files, unfortunately)